bread/layout/components/form.py

```diff
@@ -258,9 +261,10 @@ def _mapwidget(
 
```

Comment:
Good catch with the ```WIDGET_MAPPING```.
As already mentioned, this part of the code is one the more messy ones I think. But anyway, the PR changes are consistent and you were able work it out by yourself, so that is already something :)